scripts/globals/battlefield.lua

```diff
@@ -171,12 +171,17 @@ function xi.battlefield.HandleWipe(battlefield, players)
         end
 
         -- Party has wiped. Save and send time remaining before being booted.
+        -- TODO: Add LUA Binding to check for BCNM flag - RULES_REMOVE_3MIN = 0x04,
```

Comment:
Might need to make sure Xaver's recent PR doesn't always cover this

Reply:
Do you know which PR this is?

Reply:
Confirmed his stuff doesn't touch this